index.html

```diff
@@ -315,9 +315,93 @@ <h2 class="component_ex">Example of Floating action Button</h2>
                     </iframe>
                 </div>
                 <hr>
+                <!-- Image component -->
+                <div id="Image">
+                    <h1 class="component_heading">Image</h1>
+                    <p class="para">
+                        Responsive images are the set of techniques used to load the right image based on device
+                        resolution, orientation, screen size, network connection, and page layout.
+                    </p>
+                    <h2 class="component_ex">Example For Responsive Images</h2>
+                    <div>
+                        <p class="para">
+                            This image component contain responsive image you can use this with copying below mentioned
+                            html snippet.
+                            added 3 type of image like- square image ,round shape image
+                        </p>
+                    </div>
+                    <div class="component_box">
+                        <div class="sq_image">
+                            <img src="/component/image/res_image.jpg" class="image" alt="image">
+                            <img src="/component/image/res_image.jpg" class="image image_round" alt="image">
+                        </div>
```

Comment:
Can you explain this? I'm confused whether both are square images or one of them is square and the other is round. If so, why give the div a classname of sq_image.

Reply:
thank you for pointing out. I forget to change the class name as I'm using it for both images .